public/styles/site/newshub.sass

```diff
@@ -3,13 +3,20 @@ $headerColour: #6990e0
 .newshub_container
   min-height: calc(100vh)
   background-color: #212121
+  overflow: hidden
   
 .newshub
   max-width: 1270px
   margin: auto
   padding-top: 25px
   color: rgb(214, 214, 214)
-
+  position: relative
+  
+.newshub_flex
```

Comment:
Do we mix block and flex displaying?

Reply:
It will be changed in next version. Probably today.

Reply:
norraxx, this PR is just to fix the bugs that are in prod at the moment

the next release is a rewrite that offers more functionality but also addresses lots of implementation issues like not using sass nesting

can we please just get this approved and pushed out?